cli/cli/helpers/engine_manager/engine_existence_guarantor.go

```diff
@@ -132,6 +135,15 @@ func (guarantor *engineExistenceGuarantor) getPostVisitingHostMachineIpAndPort()
 func (guarantor *engineExistenceGuarantor) VisitStopped() error {
 	logrus.Infof("No Kurtosis engine was found; attempting to start one...")
 
+	if guarantor.kurtosisClusterType == resolved_config.KurtosisClusterType_Docker {
```

Comment:
Interesting. I'm wondering, does this mean the CLI is responsible for starting the centralized logging components? I think it would be much much better to have the engine own this. The CLI is just a client, for now the only one we have, but that might not always be true (think about the cloud). If we can have the engine itself own most of the logic, it will be a lot more future proof

Reply:
agreed, going to shift creation/destruction of log aggregator to engine so that it's lifecycle is one to one with the engines